Upstream-Status: Inappropriate [distribution]

rebased from original configure_madness.patch

libtasn1 provides an .pc file in upstream,
and a libgcrypt.pc is added in libgcrypt recipe,
so use Requires.private for these two.

against 2.8.6

07/02/2010 - qhe

---
Index: gnutls-2.12.5/lib/configure.ac
===================================================================
--- gnutls-2.12.5.orig/lib/configure.ac	2011-05-11 00:05:33.000000000 -0700
+++ gnutls-2.12.5/lib/configure.ac	2011-05-25 19:45:23.721180103 -0700
@@ -96,8 +96,8 @@
 dnl for pakchois
 AC_LIB_HAVE_LINKFLAGS(dl,, [#include <dlfcn.h>], [dlclose (0);])
 
-LIBGNUTLS_LIBS="-L${libdir} -lgnutls $LIBS"
-LIBGNUTLS_CFLAGS="-I${includedir}"
+LIBGNUTLS_LIBS="-lgnutls $LIBS"
+LIBGNUTLS_CFLAGS=""
 AC_SUBST(LIBGNUTLS_LIBS)
 AC_SUBST(LIBGNUTLS_CFLAGS)
 
Index: gnutls-2.12.5/lib/gnutls.pc.in
===================================================================
--- gnutls-2.12.5.orig/lib/gnutls.pc.in	2011-04-23 08:00:07.000000000 -0700
+++ gnutls-2.12.5/lib/gnutls.pc.in	2011-05-25 19:50:11.926102288 -0700
@@ -19,7 +19,6 @@
 Description: Transport Security Layer implementation for the GNU system
 URL: http://www.gnu.org/software/gnutls/
 Version: @VERSION@
+Requires.private: libgcrypt, libtasn1
 Libs: -L${libdir} -lgnutls
-Libs.private: @LTLIBGCRYPT@ @NETTLE_LIBS@
-@GNUTLS_REQUIRES_PRIVATE@
 Cflags: -I${includedir}
Index: gnutls-2.12.5/libextra/configure.ac
===================================================================
--- gnutls-2.12.5.orig/libextra/configure.ac	2011-05-11 00:05:31.000000000 -0700
+++ gnutls-2.12.5/libextra/configure.ac	2011-05-25 19:45:23.722186801 -0700
@@ -41,8 +41,8 @@
 
 xgl_INIT
 
-LIBGNUTLS_EXTRA_LIBS="-L${libdir} -lgnutls-extra $LZO_LIBS $LIBGNUTLS_LIBS"
-LIBGNUTLS_EXTRA_CFLAGS="-I${includedir}"
+LIBGNUTLS_EXTRA_LIBS="-lgnutls-extra $LZO_LIBS $LIBGNUTLS_LIBS"
+LIBGNUTLS_EXTRA_CFLAGS=""
 AC_SUBST(LIBGNUTLS_EXTRA_LIBS)
 AC_SUBST(LIBGNUTLS_EXTRA_CFLAGS)
 
Index: gnutls-2.12.5/libextra/gnutls-extra.pc.in
===================================================================
--- gnutls-2.12.5.orig/libextra/gnutls-extra.pc.in	2011-04-07 17:30:45.000000000 -0700
+++ gnutls-2.12.5/libextra/gnutls-extra.pc.in	2011-05-25 19:45:23.722186801 -0700
@@ -22,5 +22,5 @@
 Requires: gnutls
 Version: @VERSION@
 Libs: -L${libdir} -lgnutls-extra
-Libs.private: @LIBGNUTLS_EXTRA_LIBS@
+Libs.private: -llzo2
 Cflags: -I${includedir}
